doc/admin-guide/files/records.config.en.rst

```diff
@@ -3400,6 +3400,24 @@ HTTP/2 Configuration
 
    Enable the experimental HTTP/2 Stream Priority feature.
 
+.. ts:cv:: CONFIG proxy.config.http2.accept_no_activity_timeout INT 120
+   :reloadable:
+   :overridable:
+
+   Specifies how long Traffic Server keeps connections to origin servers open
+   if the transaction stalls. Lowering this timeout can ease pressure on the
+   proxy if misconfigured or misbehaving clients are opening a large number of
```

Comment:
If the first sentence is true ("how long ATS keeps connections to origin servers"), then, the number of (or slow) client connections would not be impacted by this setting, right? That is, it more or less describes and how often ats would re-establish the connection to origin.

But... perhaps that first sentence isn't correct? Because, ATS doesn't do HTTP2 to the origin -- only from client to ATS.